client/src/pages/import-students/adjust-data-form/components/StudentDataBox.helpers.ts

```diff
@@ -0,0 +1,38 @@
+import { StudentStatus } from 'shared/model/Student';
+import { CSVData, CSVMappedColumns } from '../../../../components/import-csv/ImportCSV.types';
+import { StudentColumns } from '../../ImportStudents';
+import { StudentFormState } from '../AdjustImportedStudentDataForm';
+
+interface ConversionParams {
+    data: CSVData;
+    values: CSVMappedColumns<StudentColumns>;
+}
+
+function convertColumnToStatus(statusData?: string): StudentStatus {
+    const sanitized = statusData?.trim()?.toUpperCase();
+    const status = Object.values(StudentStatus).find((status) => sanitized === status.toString());
+    return status ?? StudentStatus.ACTIVE;
+}
+
+export function convertCSVDataToFormData(params: ConversionParams): StudentFormState {
+    const { data, values } = params;
+    const emptyString = 'N/A';
+
+    const userFormState: StudentFormState = {};
+    data.rows.forEach(({ rowNr, data }) => {
+        const key = rowNr.toString();
+        userFormState[key] = {
+            rowNr,
+            firstname: data[values.firstname as string] || emptyString,
+            lastname: data[values.lastname as string] || emptyString,
+            email: data[values.email as string] || undefined,
+            status: convertColumnToStatus(data[values.status as string]),
+            team: data[values.team as string]?.trim(),
```

Comment:
Why is the team field getting trimmed as only field?

Reply:
The team is used as grouping key, I wanted to prevent whitespace to lead to the wrong grouping. For the other, I did not really bother (no harm if not trimmed), so I kept it similar to the user import. However, if you think that trimming is a good idea, I can add it everywhere.

Reply:
I see. Trimming would be useful for names and e-mails, maybe. But it is nothing that has a high priority, I think.

Reply:
so, shall I add this with this PR?
if so, should I also update it for the user import?

Reply:
If you have spare time you can add the trimming with this PR. If you don't have time you can omit it and we create an issue to add it later.

Reply:
added with cd14c97374be4c1dd2272c58a44e2dea5690d540

Reply:
added with cd14c97374be4c1dd2272c58a44e2dea5690d540